src/ia-account-settings.ts

```diff
@@ -0,0 +1,897 @@
+            <label for="email">
+              Change email <small>(verification will be required)</small>
+            </label>
+            <input
+              type="email"
+              class="form-control"
+              id="email"
+              name="email"
+              .value="${this.userData.email}"
+              @input=${this.setEmail}
+              @blur=${this.validateEmail}
+            />
+            <span class="error-field">${this.fieldsError.email}</span>
+          </div>
+
+          <div class="form-element">
+            <label for="password">
+              Change Internet Archive / Open Library password
+            </label>
+            <input
+              type="password"
+              class="form-control"
+              id="password"
+              name="password"
+              autocomplete="new-password"
+              @input=${this.setPassword}
+              @blur=${this.validatePassword}
+            />
+            <input
+              type="image"
+              class="password-icon"
+              src="https://archive.org/images/eye-crossed.svg"
+              alt="View text"
+              @click=${(e: Event) => {
+                preventDefaultAndStopEvent(e);
+                togglePassword(e, this.passwordField as HTMLInputElement);
+              }}
+            />
+            <span class="error-field">${this.fieldsError.password}</span>
+          </div>
+
+          <div class="form-element">
+            <label>Set borrow history</label>
+            <input
+              type="checkbox"
+              id="borrow-history"
+              name="borrow-history ${this.loanHistoryFlag}"
+              .checked=${this.loanHistoryFlag === 'public' ||
+              this.loanHistoryFlag === true}
+              @click=${this.setBorrowHistory}
+            />
+            <label for="borrow-history"> Visible to the public</label>
+          </div>
+
+          <div class="form-element newsletter">
+            <label>Newsletter subscriptions</label>
+            <p>
+              Stay up to date with what's happening at the Internet Archive by
+              signing up for our free newsletters.
+            </p>
+            ${this.mailingListsTemplate}
+          </div>
+
+          <div class="form-element">
+            <label for="linked-account"
+              >Linked 3rd party accounts (e.g. Google)</label
+            >
+            ${Object.keys(this.linkedProviders).length
+              ? this.linkedAccountTemplate
+              : 'You have no linked accounts'}
+          </div>
+
+          <div
+            class="form-element delete-link ${this.attemptToDelete
+              ? 'hide'
+              : ''}"
+          >
+            <a
+              href="javascript:void(0)"
+              @click=${() => {
+                this.attemptToDelete = true;
+                window.scrollTo();
+              }}
+              >Delete Internet Archive / Open Library account
+            </a>
```

Comment:
Suggestion: This looks like it should be a button, rather than a void anchor. (We can still style it to look like a link).

Also: Is the `window.scrollTo()` necessary? Without x/y coordinates, isn't that a no-op?